### PR TITLE
Add control for arm64 for package libcrystalhd-dev

### PR DESCRIPTION
**Pull request by muratugureminoglu**

### CODE REVIEW

File: install_ant-media-server.sh [hunk 1]
```diff
@@ -245,8 +245,10 @@ if [ "$ID" == "ubuntu" ]; then
   VERSION=$(unzip -p "$ANT_MEDIA_SERVER_ZIP_FILE" ant-media-server/ant-media-server.jar  | busybox unzip -p - | grep -a "Implementation-Version"|cut -d' ' -f2 | tr -d '\r')
   if [ "$(printf '%s\n' "$REQUIRED_VERSION" "$VERSION" | sort -V | head -n1)" != "$REQUIRED_VERSION" ]; then
       check_version
-      $SUDO apt-get install libcrystalhd-dev -y
-      check
+      if [ "x86_64" == $ARCH ]; then
```
[the remark this answers is not in the document]
Comment (mekya) [reply install_ant-media-server.sh:248]: @sonatype-lift ignoreall

[the remark this answers is not in the document]
Comment (mekya) [reply install_ant-media-server.sh:248]: Please try to fit the best practices

File: install_ant-media-server.sh [hunk 2]
```diff
@@ -245,8 +245,10 @@ if [ "$ID" == "ubuntu" ]; then
...
+        $SUDO apt-get install libcrystalhd-dev -y
```
Comment (mekya) [inline install_ant-media-server.sh:249]: Please add some comments why we're making this change

**Review by mekya — APPROVED** [review]
Thank you for your work. 
I've made some comments. They are not blocker. So I'm approving.

File: install_ant-media-server.sh (hunk 2 above)
Comment (muratugureminoglu) [reply install_ant-media-server.sh:270]: It 's my bad sorry. It is important to make this a habit and show more care in incorporating it consistently. :)

**Pull request merged**